spec/acceptance/ios_interface_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'spec_helper_acceptance'
+
+describe 'ios_interface' do
+  target = if ['3650', '3750', '2960'].include?(device_model)
+             'GigabitEthernet1/0'
+           elsif ['4507', '4948', '6503'].include?(device_model)
+             'GigabitEthernet1'
+           elsif ['4503'].include?(device_model)
+             'GigabitEthernet2'
+           else
+             'GigabitEthernet0'
+           end
+  it 'Set Two Instances' do
+    pp = <<-EOS
+    ios_interface { '#{target}/1':
+      link_status => true,
+    }
+    ios_interface { '#{target}/2':
+      mac_notification_added => true,
```

Comment:
As there are some devices which do not support these params then these tests will fail idempotency for the devices that do not support these fields

Reply:
Added checks for the version